In [15]:
from dbfread import DBF
import pandas as pd
from pandas import DataFrame

## Load 10 years of accident data, from 2007 to 2016

In [2]:
dbf07= DBF('accident/accident2007.dbf')
dbf08= DBF('accident/accident2008.dbf')
dbf09= DBF('accident/accident2009.dbf')
dbf10= DBF('accident/accident2010.dbf')
dbf11 = DBF('accident/accident2011.dbf')
dbf12 = DBF('accident/accident2012.dbf')
dbf13 = DBF('accident/accident2013.dbf')
dbf14 = DBF('accident/accident2014.dbf')
dbf15 = DBF('accident/accident2015.dbf')
dbf16 = DBF('accident/accident2016.dbf')
accidents07 = DataFrame(iter(dbf07))
accidents08 = DataFrame(iter(dbf08))
accidents09 = DataFrame(iter(dbf09))
accidents10 = DataFrame(iter(dbf10))
accidents11 = DataFrame(iter(dbf11))
accidents12 = DataFrame(iter(dbf12))
accidents13 = DataFrame(iter(dbf13))
accidents14 = DataFrame(iter(dbf14))
accidents15 = DataFrame(iter(dbf15))
accidents16 = DataFrame(iter(dbf16))


First, we want to combine accidents10 ~ accidents16 to one dataframe. Since not all of the accident data downloaded from the U.S. Department of Transportation have the same features, by using the `jion:inner` option in `pd.concat` function, we can get the intersection of features.

In [4]:
# rename column name in frame07 so that columns names are the same with other frames
accidents07.rename(columns={'latitude': 'LATITUDE', 'longitud': 'LONGITUD'}, inplace=True)

In [17]:
# take a look inside how the accident data file looks like
allaccidents = pd.concat([accidents07,accidents08,accidents09,accidents10,accidents11,accidents12,accidents13,accidents14,accidents15,accidents16], axis=0,join='inner')
pd.set_option('display.max_columns', 100)
allaccidents.head()

STATE  COUNTY  MONTH   DAY  HOUR  MINUTE  VE_TOTAL  PERSONS  PEDS  NHS  \
0    1.0    73.0    1.0   2.0  23.0    15.0       1.0      1.0   0.0  0.0   
1    1.0    19.0    1.0  30.0  13.0     5.0       3.0      4.0   0.0  0.0   
2    1.0     9.0    1.0  14.0  12.0    13.0       2.0      5.0   0.0  0.0   
3    1.0    15.0    1.0   1.0  15.0     0.0       2.0      2.0   0.0  1.0   
4    1.0    51.0    1.0   6.0  19.0    35.0       2.0      5.0   0.0  0.0   

   ROUTE  SP_JUR  HARM_EV  MAN_COLL  REL_ROAD  LGT_COND  WEATHER1  WEATHER2  \
0    4.0     0.0     42.0       0.0       4.0       2.0       1.0       0.0   
1    3.0     0.0     12.0       5.0       1.0       1.0       1.0       0.0   
2    2.0     0.0     12.0       7.0       1.0       1.0       1.0       0.0   
3    2.0     0.0     12.0       4.0       1.0       1.0       1.0       0.0   
4    3.0     0.0     12.0       5.0       1.0       3.0       1.0       0.0   

   NOT_HOUR  NOT_MIN  ARR_HOUR  ARR_MIN  HOSP_HR  HOSP_MN  SCH_BUS  CF1  CF2  \
0      23.0     16.0      23.0     20.0      0.0      0.0      0.0  0.0  0.0   
1      13.0     10.0      13.0     11.0     99.0     99.0      0.0  0.0  0.0   
2      12.0     16.0      12.0     18.0     99.0     99.0      0.0  0.0  0.0   
3      99.0     99.0      15.0      5.0      0.0      0.0      0.0  0.0  0.0   
4      19.0     35.0      19.0     41.0     99.0     99.0      0.0  0.0  0.0   

   CF3  FATALS  DAY_WEEK  DRUNK_DR  ST_CASE    CITY  MILEPT    YEAR  TWAY_ID  \
0  0.0     1.0       3.0       0.0  10001.0     0.0     0.0  2007.0     1493   
1  0.0     2.0       3.0       0.0  10002.0   630.0     0.0  2007.0    SR-68   
2  0.0     1.0       1.0       0.0  10003.0  2330.0     0.0  2007.0  US-SR53   
3  0.0     1.0       2.0       0.0  10004.0     0.0  2468.0  2007.0   US-SR1   
4  0.0     1.0       7.0       0.0  10005.0  2063.0     0.0  2007.0    SR-14   

   TWAY_ID2     RAIL  VE_FORMS  WEATHER   LATITUDE   LONGITUD  
0  00000000  0000000       1.0      1.0  33.455839 -87.017928  
1       270  0000000       3.0      1.0  34.160597 -85.678075  
2  00000000  0000000       2.0      1.0  33.975717 -86.496347  
3      7260  0000000       2.0      1.0  33.853258 -85.915622  
4      I-65  0000000       2.0      1.0  32.488833 -86.408936

In [ ]:
# allaccidents['ST_CASE'] = 'n' + df['ST_CASE']

In [7]:
allaccidents.shape

(320874, 42)

The allaccidents table recorded 320874 accidents from 2010-2016, and it has 42 features. Here are the meaning of some of the features according to the `FARS Analytical User’s Manual`.

### Explaination of variables
*VE_TOTAL*: Number of Vehicle in crash <br/>
*VE_FORMS*: Number of Motor Vehicles in Transport (MVIT) <br/>
*PED*: Number of Persons Not in Motor Vehicles <br/>
*NHS*: National Highway System<br/>
*ROUTE*: Route Signing <br/>
*SP_JUR*: Special Jurisdiction <br/>
*HARM_EV*: First Harmful Event<br/>
*TWAY_ID , TWAY_ID2* : Trafficway Identifier <br/>
*MILEPT*: Milepoint <br/>
*SP_JUR*: Special Jurisdiction<br/>
*HARM_EV*: injury or damage producing First Harmful Event <br/> 
*MAN_COLL*:Manner of Collision  <br/> 
*RELJCT1, RELJCT2*: Relation to Junction- Within Interchange Area, Specific Location. <br/>
*TYP_INT*: Type of Intersection <br/>
*REL_ROAD*: Relation to Trafficway <br/>
*LGT_COND*: Light Condition<br/> 
*NOT_HOUR,MIN*: Min, Hour of Notification <br/>
*ARR_HOUR,MIN*: Hour, Min arrival at scene <br/>
*HOSP_HR,MIN*: Hour, Min arrival at hospital <br/>
*CF1, CF2, CF3*:Related Factors- Crash Level, factors related to the crash <br/>
*FATALS*: Fatalities<br/>
*DRUNK_DR*: Number of Drinking Drivers<br/> 
*RAIL*: Rail Grade Crossing Identifier<br/>

For more detailed information, please refer to `FARS Analytical User’s Manual`.

## Select variables and rename variables

Observed from the table above, some of the variables in the table are not very readable. Therefore, in order to make it easier to understand the variables,we renamed some of the variables according to `FARS Analytical User’s Manual`  downloaded from the  `U.S. Department of Transportation`  website.

### Load vehicle data file which contains mortality rate

We also want to study the mortality rate of fatal accidents. The data element “Fatalities in Vehicle” in the Vehicle data file from the `U.S. Department of Transportation` website provides the number of deaths in a vehicle.

In [27]:
vdbf07= DBF('vehicle_deaths/vehicle2007.dbf')
vdbf08= DBF('vehicle_deaths/vehicle2008.dbf')
vdbf09= DBF('vehicle_deaths/vehicle2009.dbf')
vdbf10= DBF('vehicle_deaths/vehicle2010.dbf')
vdbf11= DBF('vehicle_deaths/vehicle2011.dbf')
vdbf12= DBF('vehicle_deaths/vehicle2012.dbf')
vdbf13= DBF('vehicle_deaths/vehicle2013.dbf')
vdbf14= DBF('vehicle_deaths/vehicle2014.dbf')
# vdbf15= DBF('vehicle_deaths/vehicle2015.csv')
vdbf16= DBF('vehicle_deaths/vehicle2016.dbf')
vehicle07 = DataFrame(iter(vdbf07))
vehicle08 = DataFrame(iter(vdbf08))
vehicle09 = DataFrame(iter(vdbf09))
vehicle10 = DataFrame(iter(vdbf10))
vehicle11 = DataFrame(iter(vdbf11))
vehicle12 = DataFrame(iter(vdbf12))
vehicle13 = DataFrame(iter(vdbf13))
vehicle14 = DataFrame(iter(vdbf14))
# vehicle15 = pd.read_csv('vehicle_deaths/vehicle2015.csv')
vehicle16 = DataFrame(iter(vdbf16))

In [31]:
vehicle07['YEAR']='2007.0'
vehicle08['YEAR']='2008.0'
vehicle09['YEAR']='2009.0'
vehicle10['YEAR']='2010.0'
vehicle11['YEAR']='2011.0'
vehicle12['YEAR']='2012.0'
vehicle13['YEAR']='2013.0'
vehicle14['YEAR']='2014.0'
# vehicle15['YEAR']='2015.0'
vehicle16['YEAR']='2016.0'

In [32]:
#combine all vehicle file
allvehicle = pd.concat([vehicle07,vehicle08,vehicle09,vehicle10,vehicle11,vehicle12,vehicle13,vehicle14,vehicle16], axis=0,join='inner')
allvehicle.head(10)

VIN VIN_1 VIN_2 VIN_3 VIN_4 VIN_5 VIN_6 VIN_7 VIN_8 VIN_9 VIN_10  \
0  JHMCB7657NC0     J     H     M     C     B     7     6     5     7      N   
1  JN1HJ01P7LT4     J     N     1     H     J     0     1     P     7      L   
2  1G6CD53BXP42     1     G     6     C     D     5     3     B     X      P   
3  1FTPW12524KB     1     F     T     P     W     1     2     5     2      4   
4  1D4HD58266F1     1     D     4     H     D     5     8     2     6      6   
5  2BCCV81J7JB5     2     B     C     C     V     8     1     J     7      J   
6  1G2NE52T5WM5     1     G     2     N     E     5     2     T     5      W   
7  1FTZX17W1WNB     1     F     T     Z     X     1     7     W     1      W   
8  1FAFP40604F1     1     F     A     F     P     4     0     6     0      4   
9  1GHDX03E6XD2     1     G     H     D     X     0     3     E     6      X   

  VIN_11 VIN_12  STATE  VEH_NO  MAKE  BODY_TYP  REG_STAT  OWNER  ROLLOVER  \
0      C      0    1.0     1.0  37.0       4.0       1.0    2.0       0.0   
1      T      4    1.0     1.0  35.0       4.0       1.0    1.0       0.0   
2      4      2    1.0     2.0  19.0       4.0       1.0    1.0       0.0   
3      K      B    1.0     3.0  12.0      31.0       1.0    1.0       0.0   
4      F      1    1.0     1.0   7.0      15.0       1.0    1.0       0.0   
5      B      5    1.0     2.0   2.0      14.0       1.0    2.0       2.0   
6      M      5    1.0     1.0  22.0       4.0       1.0    1.0       0.0   
7      N      B    1.0     2.0  12.0      31.0       1.0    1.0       0.0   
8      F      1    1.0     1.0  12.0       2.0       1.0    2.0       0.0   
9      D      2    1.0     2.0  21.0      20.0       1.0    1.0       0.0   

   J_KNIFE  TRAV_SP  HAZ_INV  HAZ_PLAC  HAZ_CNO  HAZ_REL  TOW_VEH  V_CONFIG  \
0      0.0     65.0      1.0       0.0      0.0      0.0      0.0       0.0   
1      0.0     10.0      1.0       0.0      0.0      0.0      0.0       0.0   
2      0.0     55.0      1.0       0.0      0.0      0.0      0.0       0.0   
3      0.0     55.0      1.0       0.0      0.0      0.0      0.0       0.0   
4      1.0     55.0      1.0       0.0      0.0      0.0      1.0       0.0   
5      0.0     50.0      1.0       0.0      0.0      0.0      0.0       0.0   
6      0.0     20.0      1.0       0.0      0.0      0.0      0.0       0.0   
7      0.0     62.0      1.0       0.0      0.0      0.0      0.0       0.0   
8      0.0     15.0      1.0       0.0      0.0      0.0      0.0       0.0   
9      0.0     50.0      1.0       0.0      0.0      0.0      0.0       0.0   

   CARGO_BT  SPEC_USE  EMER_USE  IMPACT1  UNDERIDE  DEFORMED  FIRE_EXP  \
0       0.0       0.0       0.0      9.0       0.0       6.0       0.0   
1       0.0       0.0       0.0     12.0       0.0       6.0       0.0   
2       0.0       0.0       0.0      7.0       0.0       6.0       0.0   
3       0.0       0.0       0.0     12.0       0.0       6.0       0.0   
4       0.0       0.0       0.0      5.0       0.0       2.0       0.0   
5       0.0       0.0       0.0      7.0       0.0       6.0       0.0   
6       0.0       0.0       0.0      9.0       0.0       6.0       0.0   
7       0.0       0.0       0.0     12.0       0.0       6.0       0.0   
8       0.0       0.0       0.0      3.0       0.0       6.0       0.0   
9       0.0       0.0       0.0     12.0       0.0       6.0       0.0   

   M_HARM  DEATHS  BUS_USE  GVWR  UNITTYPE  MCARR_I1  ST_CASE  HAZ_ID  \
0    42.0     1.0      0.0   0.0       1.0       0.0  10001.0     0.0   
1    12.0     0.0      0.0   0.0       1.0       0.0  10002.0     0.0   
2    12.0     2.0      0.0   0.0       1.0       0.0  10002.0     0.0   
3    12.0     0.0      0.0   0.0       1.0       0.0  10002.0     0.0   
4    12.0     0.0      0.0   0.0       1.0       0.0  10003.0     0.0   
5    12.0     1.0      0.0   0.0       1.0       0.0  10003.0     0.0   
6    12.0     1.0      0.0   0.0       1.0       0.0  10004.0     0.0   
7    12.0     

In [33]:
mortalities = allvehicle[['ST_CASE','YEAR','DEATHS']]
mortalities.head()

ST_CASE    YEAR  DEATHS
0  10001.0  2007.0     1.0
1  10002.0  2007.0     0.0
2  10002.0  2007.0     2.0
3  10002.0  2007.0     0.0
4  10003.0  2007.0     0.0

In [37]:
mortality_cleaned = mortalities.groupby(['ST_CASE','YEAR']).sum().reset_index()
mortality_cleaned.head()

ST_CASE    YEAR  DEATHS
0  10001.0  2007.0     1.0
1  10001.0  2008.0     0.0
2  10001.0  2009.0     1.0
3  10001.0  2010.0     1.0
4  10001.0  2011.0     1.0

In [20]:
allaccidents['DEATHS']= allaccidents.ST_CASE.map(mortality_cleaned.DEATHS)
allaccidents.head()

STATE  COUNTY  MONTH   DAY  HOUR  MINUTE  VE_TOTAL  PERSONS  PEDS  NHS  \
0    1.0    73.0    1.0   2.0  23.0    15.0       1.0      1.0   0.0  0.0   
1    1.0    19.0    1.0  30.0  13.0     5.0       3.0      4.0   0.0  0.0   
2    1.0     9.0    1.0  14.0  12.0    13.0       2.0      5.0   0.0  0.0   
3    1.0    15.0    1.0   1.0  15.0     0.0       2.0      2.0   0.0  1.0   
4    1.0    51.0    1.0   6.0  19.0    35.0       2.0      5.0   0.0  0.0   

   ROUTE  SP_JUR  HARM_EV  MAN_COLL  REL_ROAD  LGT_COND  WEATHER1  WEATHER2  \
0    4.0     0.0     42.0       0.0       4.0       2.0       1.0       0.0   
1    3.0     0.0     12.0       5.0       1.0       1.0       1.0       0.0   
2    2.0     0.0     12.0       7.0       1.0       1.0       1.0       0.0   
3    2.0     0.0     12.0       4.0       1.0       1.0       1.0       0.0   
4    3.0     0.0     12.0       5.0       1.0       3.0       1.0       0.0   

   NOT_HOUR  NOT_MIN  ARR_HOUR  ARR_MIN  HOSP_HR  HOSP_MN  SCH_BUS  CF1  CF2  \
0      23.0     16.0      23.0     20.0      0.0      0.0      0.0  0.0  0.0   
1      13.0     10.0      13.0     11.0     99.0     99.0      0.0  0.0  0.0   
2      12.0     16.0      12.0     18.0     99.0     99.0      0.0  0.0  0.0   
3      99.0     99.0      15.0      5.0      0.0      0.0      0.0  0.0  0.0   
4      19.0     35.0      19.0     41.0     99.0     99.0      0.0  0.0  0.0   

   CF3  FATALS  DAY_WEEK  DRUNK_DR  ST_CASE    CITY  MILEPT    YEAR  TWAY_ID  \
0  0.0     1.0       3.0       0.0  10001.0     0.0     0.0  2007.0     1493   
1  0.0     2.0       3.0       0.0  10002.0   630.0     0.0  2007.0    SR-68   
2  0.0     1.0       1.0       0.0  10003.0  2330.0     0.0  2007.0  US-SR53   
3  0.0     1.0       2.0       0.0  10004.0     0.0  2468.0  2007.0   US-SR1   
4  0.0     1.0       7.0       0.0  10005.0  2063.0     0.0  2007.0    SR-14   

   TWAY_ID2     RAIL  VE_FORMS  WEATHER   LATITUDE   LONGITUD  DEATHS  
0  00000000  0000000       1.0      1.0  33.455839 -87.017928     4.0  
1       270  0000000       3.0      1.0  34.160597 -85.678075     2.0  
2  00000000  0000000       2.0      1.0  33.975717 -86.496347     2.0  
3      7260  0000000       2.0      1.0  33.853258 -85.915622     2.0  
4      I-65  0000000       2.0      1.0  32.488833 -86.408936     3.0

In [52]:
accidents = allaccidents[['YEAR','ST_CASE','STATE','VE_TOTAL','PERSONS','FATALS','DEATHS','MONTH','DAY_WEEK','HOUR','NHS','LATITUDE','LONGITUD','MAN_COLL','LGT_COND','WEATHER','ARR_HOUR','ARR_MIN','CF1','DRUNK_DR']]
accidents.rename(columns={'ST_CASE':'CASE_NUM','VE_TOTAL':'NUM_VEHICLE','NHS': 'HIGHWAY', 'MAN_COLL': 'COLLISION_TYPE','LGT_COND':'LIGHT_CONDITION','CF1':'CRASH_FACTOR','DRUNK_DR':'DRUNK_DRIVE'}, inplace=True)
accidents['MONTH'] = accidents['MONTH'].map({1.0:'January', 2.0:'February', 3.0: 'March', 4.0:'April', 5.0:'May', 6.0:'June', 7.0:'July', 8.0:'August',9.0: 'September', 10.0:'October', 11.0:'November', 12.0:'December'})
accidents['DAY_WEEK']= accidents['DAY_WEEK'].map({1.0:'Sunday',2.0:'Monday', 3.0:'Tuesday', 4.0: 'Wednesday', 5.0:'Thursday', 6.0:'Friday', 7.0:'Saturday'})
accidents

/Users/aitingkuang/anaconda/lib/python3.6/site-packages/pandas/core/frame.py:2844: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
  **kwargs)
/Users/aitingkuang/anaconda/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
  This is separate from the ipykernel package so we can avoid doing imports until
/Users/aitingkuang/anaconda/lib/python3.6/site-packages/ipykernel_launcher.py:4: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the docu

YEAR  CASE_NUM  STATE  NUM_VEHICLE  PERSONS  FATALS  DEATHS  \
0      2007.0   10001.0    1.0          1.0      1.0     1.0     4.0   
1      2007.0   10002.0    1.0          3.0      4.0     2.0     2.0   
2      2007.0   10003.0    1.0          2.0      5.0     1.0     2.0   
3      2007.0   10004.0    1.0          2.0      2.0     1.0     2.0   
4      2007.0   10005.0    1.0          2.0      5.0     1.0     3.0   
5      2007.0   10006.0    1.0          1.0      2.0     1.0     3.0   
6      2007.0   10007.0    1.0          2.0      2.0     1.0     1.0   
7      2007.0   10008.0    1.0          2.0      2.0     1.0     3.0   
8      2007.0   10009.0    1.0          2.0      7.0     1.0     0.0   
9      2007.0   10010.0    1.0          2.0      2.0     1.0     3.0   
10     2007.0   10011.0    1.0          1.0      2.0     1.0     3.0   
11     2007.0   10012.0    1.0          3.0      5.0     1.0     3.0   
12     2007.0   10013.0    1.0          1.0      1.0     1.0     3.0   
13     2007.0   10014.0    1.0          1.0      1.0     1.0     4.0   
14     2007.0   10015.0    1.0          2.0      3.0     1.0     0.0   
15     2007.0   10016.0    1.0          1.0      2.0     2.0     2.0   
16     2007.0   10017.0    1.0          1.0      1.0     1.0     1.0   
17     2007.0   10018.0    1.0          1.0      1.0     1.0     1.0   
18     2007.0   10019.0    1.0          2.0      2.0     1.0     1.0   
19     2007.0   10020.0    1.0          2.0      3.0     1.0     2.0   
20     2007.0   10021.0    1.0          3.0      3.0     1.0     3.0   
21     2007.0   10022.0    1.0          2.0      2.0     1.0     3.0   
22     2007.0   10023.0    1.0          1.0      1.0     1.0     1.0   
23     2007.0   10024.0    1.0          1.0      1.0     1.0     3.0   
24     2007.0   10025.0    1.0          1.0      2.0     1.0     2.0   
25     2007.0   10026.0    1.0          3.0      4.0     1.0     3.0   
26     2007.0   10027.0    1.0          2.0      3.0     1.0     2.0   
27     2007.0   10028.0    1.0          1.0      1.0     1.0     1.0   
28     2007.0   10029.0    1.0          2.0      2.0     1.0     1.0   
29     2007.0   10030.0    1.0          2.0      5.0     1.0     2.0   
...       ...       ...    ...          ...      ...     ...     ...   
34409  2016.0  560071.0   56.0          2.0      3.0     2.0     NaN   
34410  2016.0  560072.0   56.0          1.0      1.0     1.0     NaN   
34411  2016.0  560073.0   56.0          2.0      4.0     1.0     NaN   
34412  2016.0  560074.0   56.0          1.0      2.0     1.0     NaN   
34413  2016.0  560075.0   56.0          1.0      3.0     1.0     NaN   
34414  2016.0  560076.0   56.0          1.0      1.0     1.0     NaN   
34415  2016.0  560077.0   56.0          1.0      1.0     1.0     NaN   
34416  2016.0  560078.0   56.0          2.0      3.0     1.0     NaN   
34417  2016.0  560079.0   56.0          2.0      2.0     1.0     NaN   
34418  2016.0  560080.0   56.0          1.0      1.0     1.0     NaN   
34419  2016.0  560081.0   56.0          2.0      3.0     1.0     NaN   
34420  2016.0  560082.0   56.0          1.0      1.0     1.0     NaN   
34421  2016.0  560083.0   56.0          3.0      3.0     1.0     NaN   
34422  2016.0  560084.0   56.0          1.0      1.0     1.0     NaN   
34423  2016.0  560085.0   56.0          1.0      1.0     1.0     NaN   
34424  2016.0  560086.0   56.0          1.0      1.0     1.0     NaN   
34425  2016.0  560087.0   56.0          1.0      1.0     1.0     NaN   
34426  2016.0  560088.0   56.0          1.0      1.0     1.0     NaN   
34427  2016.0  560089.0   56.0          2.0      2.0     1.0     NaN   
34428  2016.0  560090.0   56.0          1.0      1.0     1.0     NaN   
34429  2016.0  560091.0   56.0          1.0      1.0     1.0     NaN   
34430  2016.0  560092.0   56.0          2.0      5.0     2.0     NaN   
34431  2016.0  560093.0   56.0          1.0      2.0     1.0     NaN   
34432  2016.0  560094.0   56.0          2.0      

In [45]:
print(accidents['DEATHS'])

0        4.0
1        2.0
2        2.0
3        2.0
4        3.0
5        3.0
6        1.0
7        3.0
8        0.0
9        3.0
10       3.0
11       3.0
12       3.0
13       4.0
14       0.0
15       2.0
16       1.0
17       1.0
18       1.0
19       2.0
20       3.0
21       3.0
22       1.0
23       3.0
24       2.0
25       3.0
26       2.0
27       1.0
28       1.0
29       2.0
        ... 
34409    NaN
34410    NaN
34411    NaN
34412    NaN
34413    NaN
34414    NaN
34415    NaN
34416    NaN
34417    NaN
34418    NaN
34419    NaN
34420    NaN
34421    NaN
34422    NaN
34423    NaN
34424    NaN
34425    NaN
34426    NaN
34427    NaN
34428    NaN
34429    NaN
34430    NaN
34431    NaN
34432    NaN
34433    NaN
34434    NaN
34435    NaN
34436    NaN
34437    NaN
34438    NaN
Name: DEATHS, Length: 320874, dtype: float64
